tsconfig.json

```diff
@@ -19,9 +19,11 @@
             "dom",
             "es2020"
         ],
+        "moduleResolution": "node",
```

Comment:
OOI, how does the "new" default mode break resolution?

Reply:
> OOI

Ocean Observatories Initiative? :sweat_smile: 

To be honest, I'm not exactly sure.  https://www.typescriptlang.org/docs/handbook/modules/theory.html#module-resolution indicates that the default value of the `moduleResolution` argument depends on a lot of other factors, and changing the target is surely impacting one of those.

Since we're not actually using `tsc` to emit code, I kinda don't care too much about this stuff.  I just want the one that makes it work :)

Reply:
"out of interest", is that really not well-known? :see_no_evil: 

Yeah, I don't care much either, I was just curious.